testutil/configurator/configurator.go

```diff
@@ -162,15 +160,6 @@ func AuthModule() ModuleOption {
 	}
 }
 
-func ParamsModule() ModuleOption {
```

Comment:
We could keep this one, as it brings no dep on the SDK and still useful for those testing

Reply:
i can add it back, but we already have a pr pulling it out so a new dep will come with it